src/lib.rs

```diff
@@ -207,18 +217,27 @@ A: Sized + ApplicationBase<gfx_device_dx11::Resources, D3D11CommandBuffer>
     let mut running = true;
     while running {
         let mut new_size = None;
-        events_loop.poll_events(|winit::Event::WindowEvent{window_id: _, event}| {
-            match event {
-                winit::WindowEvent::Closed => running = false,
-                winit::WindowEvent::KeyboardInput(winit::ElementState::Pressed, _, key, _) if key == A::get_exit_key() => return,
-                winit::WindowEvent::Resized(width, height) => {
-                    let size = (width as gfx::texture::Size, height as gfx::texture::Size);
-                    if size != window.size {
-                        // working around the borrow checker: window is already borrowed here
-                        new_size = Some(size);
-                    }
-                },
-                _ => app.on(event),
+        events_loop.poll_events(|event| {
```

Comment:
Code had to be reworked to handle the [`winit::Event`](https://docs.rs/winit/0.7.2/winit/enum.Event.html) rework. Events seem to be split into Device & Window now, plus in general all winit events are more nested and have switched to named variable enums.

I've simply reworked the existing code into the new structure.